addon/components/sticky-element.js

```diff
@@ -217,11 +216,39 @@ export default Component.extend({
     }
   }),
 
+  /**
+   * Add listener to update sticky element width on resize event
+   * @method initResizeEventListener
+   * @private
+   */
+  initResizeEventListener() {
+    window.addEventListener('resize', this.debouncedUpdateDimension.bind(this), false);
+  },
+
+  /**
+   * @method removeResizeEventListener
+   * @private
+   */
+  removeResizeEventListener() {
+    window.removeEventListener('resize', this.debouncedUpdateDimension.bind(this), false);
```

Comment:
I *believe* this is not really working, i.e. leaving an event listener around.`.bind()` will create a new function instance everytime you call it, and `removeEventListener()` will only work when you pass the same function instance you passed to `addEventListener()`;

I usually "save" the instance to some private property:
```js
initResizeEventListener() {
  this._resizeListener = () => this.debouncedUpdateDimension();
  window.addEventListener('resize', this._resizeListener, false);
},

removeResizeEventListener() {
  window.removeEventListener('resize', this._resizeListener, false);
}
```

Reply:
I've updated the event handling, and i have a performance improvement.
Cf. https://github.com/kaliber5/ember-sticky-element/pull/46/commits/7bab4bb484f5f633465e4a7831f76ea087436327